### ISSUE TITLE
Use Vortice.Windows instead of SharpDX

### ISSUE THREAD

**Issue by snaulX** [Syncaidius/MoltenEngine#19]
SharpDX is not support and not developing project. But we have alternative: [Vortice.Windows](https://github.com/amerkoleci/Vortice.Windows). This library is actively supporting and developing right now. So I advice you using Vortice.Windows instead of depricated SharpDX.

**Comment by amerkoleci** [comment]
I'm the author of Vortice.Windows, if you have any question just ask me :)

**Comment by Syncaidius** [comment]
@snaulX @amerkoleci  Thank you both. I will take a look this weekend 👍

**Comment by Syncaidius** [comment]
@snaulX I've spent this weekend checking out Vortice.Windows and it's a great library, but I've decided to go with [Silk.NET](https://github.com/dotnet/Silk.NET) as provides bindings for DirectX, OpenGL, OpenAL, OpenCL, OpenXR, GLFW, SDL, Vulkan and Assimp. I believe this will help to keep the number of dependencies a reasonable amount in Molten, as the intention is to develop it as a cross-platform game engine. 

@amerkoleci Keep up the awesome work with Vortice.Windows!
